subsys/mgmt/mcumgr/lib/cmd/img_mgmt/Kconfig

```diff
@@ -0,0 +1,81 @@
+# Copyright Runtime.io 2018. All rights reserved.
+# Copyright Nordic Semiconductor ASA 2020-2022. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig MCUMGR_CMD_IMG_MGMT
+	bool "Mcumgr handlers for image management"
+	select FLASH
+	select MPU_ALLOW_FLASH_WRITE if ARM_MPU
+	select IMG_MANAGER
+	help
+	  Enables mcumgr handlers for image management
+
+if MCUMGR_CMD_IMG_MGMT
+config IMG_MGMT_UL_CHUNK_SIZE
+	int "Maximum chunk size for image uploads"
+	default 512
+	depends on MCUMGR_CMD_IMG_MGMT
+	help
+	  Limits the maximum chunk size for image uploads, in bytes.  A buffer of
+	  this size gets allocated on the stack during handling of a image upload
+	  command.
+
+config IMG_MGMT_UPDATABLE_IMAGE_NUMBER
+	int "Number of supported images"
+	default UPDATEABLE_IMAGE_NUMBER
+	range 1 2
+	help
+	  Sets how many application images are supported (pairs of secondary and primary slots).
+	  Setting this to 2 requires MCUMGR_BUF_SIZE to be at least 512b.
+	  NOTE: The UPDATEABLE_IMAGE_NUMBER of MCUBOOT configuration, even for Zephyr build,
+	  needs to be set to the same value; this is due to the fact that the mcumgr uses
+	  boot_util and the UPDATEABLE_IMAGE_NUMBER controls number of images supported
+	  by that library.
+
+config IMG_MGMT_VERBOSE_ERR
+	bool "Verbose logging when uploading a new image"
+	select MGMT_VERBOSE_ERR_RESPONSE
+	help
+	  Enable verbose logging during a firmware upgrade.
+
+config IMG_MGMT_DUMMY_HDR
+	bool "Return dummy image header data for imgr functions"
+	help
+	  Returns dummy image header data for imgr functions,
+	  useful when there are no images present, Eg: unit tests.
+
+config IMG_MGMT_DIRECT_IMAGE_UPLOAD
+	bool "Allow direct image upload"
+	help
+	  Enables directly uploading image to selected image partition.
+	  This changes how "image" is understood by mcumgr: normally the mcumgr allows to upload
+	  to first slot of the only image it knows, where image is understood as two slots
+	  (two DTS images for Zephyr); this allows to treat every DTS defined image as direct
+	  target for upload, and more than two may be used (4 at this time).
+	  NOTE: When direct upload is used the image numbers are shifted by + 1, and the default
+	  behaviour is, when image is not selected, to upload to image that represents secndary
```

Comment:
*secondary

Reply:
I do not even know hot did that letter disappeared.